### PR TITLE
Dylan merchant invoices us14

### LINKED ISSUE #25: User Story 14 - Merchant Invoice

### PR DESCRIPTION
**Pull request by DylanScotty**
## Description

This will add functionality to the invoice index page
Adds test for invoice index
Adds test for item to be shipped method in merchant model spec 
Adds testing refactor in merchant model spec creating a before each

## Type of change

- [ ] fix
- [x] feat
- [x] test
- [x] refactor
- [ ] docs

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] thorough testing has been implemented if adding feature
- [x] all tests pass

### Thanks!

### CODE REVIEW

File: spec/features/merchant/invoices/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+require 'rails_helper'
```
Comment (EVanGorkom) [inline spec/features/merchant/invoices/index_spec.rb]: You're requiring 'rails_helper' twice here.

**Review by EVanGorkom — APPROVED** [review]
This looks good. I see that you added a couple new items, customers, etc. for doing sad path testing as well. Besides the extra 'rails_helper' on line 3, it looks good. How was your simplecov coverage?

File: spec/features/merchant/invoices/index_spec.rb (hunk 1 above)
Comment (DylanScotty) [reply spec/features/merchant/invoices/index_spec.rb]: Nice catch - just fixed

**Comment by DylanScotty** [comment]
coverage 100% across models and features

**Review by lanebret85 — APPROVED** [review]
Looks great, Dylan! Nice work!

**Pull request merged**